### PR TITLE
docs: add example for:awscc_ec2_volume

### PR DESCRIPTION
**Pull request by tamilver-amzn**
docs - awscc_ec2_volume

### CODE REVIEW

File: docs/resources/ec2_volume.md [hunk 1]
```diff
@@ -10,7 +9,20 @@ description: |-
 
 Resource Type definition for AWS::EC2::Volume
 
+## Example Usage
 
+### usage
+To create a volume with tags
```
[the remark this answers is not in the document]
Comment (tamilver-amzn) [reply docs/resources/ec2_volume.md]: Modified

**Comment by rahul-amzn** [comment]
LGTM

File: examples/resources/awscc_ec2_volume/volume_basic.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+resource "awscc_ec2_volume" "example" {
+  availability_zone = "us-west-2a"
+  size              = 40
+
+  tags = [
+    {
+      key =  "Name"
+      value = "HelloWorld"
+    }
+  ]
```
Comment (wellsiau-aws) [inline examples/resources/awscc_ec2_volume/volume_basic.tf:10]: this tag is not relevant, I suggest to remove it

File: templates/resources/ec2_volume.md.tmpl [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### usage
```
Comment (wellsiau-aws) [inline templates/resources/ec2_volume.md.tmpl:14]: this subsection is too generic, try to add short sentence that explain the example

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
may I also suggest to add another example for:
1. How to use KMS encryption: https://registry.terraform.io/providers/hashicorp/awscc/latest/docs/resources/ec2_volume#kms_key_id
2. Create volume from snapshot 

These addition can be in form of future pull request, thanks!

File: examples/resources/awscc_ec2_volume/volume_basic.tf [hunk 4]
```diff
@@ -0,0 +1,8 @@
+resource "awscc_ec2_volume" "example" {
+  availability_zone = "us-west-2a"
+  size              = 40
+
+  tags = {
```
[the remark this answers is not in the document]
Comment (tamilver-amzn) [reply examples/resources/awscc_ec2_volume/volume_basic.tf]: Updated similar tags

File: examples/resources/awscc_ec2_volume/volume_basic.tf (hunk 2 above)
Comment (tamilver-amzn) [reply examples/resources/awscc_ec2_volume/volume_basic.tf]: Modified the  tag values

File: templates/resources/ec2_volume.md.tmpl (hunk 3 above)
Comment (tamilver-amzn) [reply templates/resources/ec2_volume.md.tmpl]: Updated the changes. Now Kept only example usage and mentioned about the resources.

**Review by wellsiau-aws — APPROVED** [review]
LGTM

**Pull request merged**